### PR TITLE
USHIFT-4686: Add greenboot healthcheck after reboot on ostree system

### PR DESCRIPTION
**Pull request by agullon**
Jira ticket: https://issues.redhat.com/browse/USHIFT-4686

### CODE REVIEW

File: test/resources/microshift-process.resource [hunk 1]
```diff
@@ -45,6 +46,11 @@ Wait For MicroShift
     # greenboot check script to let us use it even when not on a
     # greenboot host?
 
+Wait For Greenboot Health Check To Exit
```
Comment (ggiguash) [inline test/resources/microshift-process.resource:49]: This is already defined at https://github.com/openshift/microshift/blob/main/test/resources/ostree-health.resource#L10

File: test/suites/network/offline.robot [hunk 2]
```diff
@@ -7,6 +7,7 @@ Library             ../../resources/qemu-guest-agent.py
 Library             ../../resources/DataFormats.py
 Resource            ../../resources/libvirt.resource
 Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/microshift-process.resource
```
Comment (ggiguash) [inline test/suites/network/offline.robot:10]:
```suggestion
Resource            ../../resources/ostree-health.resource
```
And use `Wait Until Greenboot Health Check Exited` function.

File: test/suites/standard1/kustomize.robot [hunk 3]
```diff
@@ -267,6 +267,7 @@ Restore Default Config
     ${is_ostree}=    Is System OSTree
     IF    ${is_ostree}
         Reboot MicroShift Host
+        Wait For Greenboot Health Check To Exit
         Wait For MicroShift
```
Comment (ggiguash) [inline test/suites/standard1/kustomize.robot:271]: No need to Wait for MicroShift if we're checking Greenboot exit.

File: test/resources/microshift-process.resource (hunk 1 above)
Comment (agullon) [reply test/resources/microshift-process.resource]: thanks, addressed in the last commit

File: test/suites/network/offline.robot (hunk 2 above)
Comment (agullon) [reply test/suites/network/offline.robot]: thanks, addressed in the last commit

File: test/suites/standard1/kustomize.robot (hunk 3 above)
Comment (agullon) [reply test/suites/standard1/kustomize.robot]: thanks, addressed in the last commit

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**